### DISCUSSION TITLE
Data with icustay_id was recorded before icu intime

### DISCUSSION THREAD

**Discussion by ld213Lin-Duan** [MIT-LCP/mimic-code#1136] [MIMIC-General]
### Description

Hi,
I'm trying to get the data during icu stay, but i found that some data with icustay_id is before the icu intime, which means the data was recorded before icu stay, but it has the icustay_id. For example, FiO2 has the charttime 2021-8-29-00:00:00, and icustay_id=a, but the intime of a is 2021-9-20-00:01:00. Was the intime recorded wrongly?

**Comment by SichengH** [comment]
In your example, most likely that FiO2 is measured before that particular ICU which is completely normal. In the future, we recommend publishing questions like this in the discussions.
